packages/admin-ui/src/components/Filter/components/Popover.tsx

```diff
@@ -0,0 +1,32 @@
+import { jsx } from '@vtex/admin-ui-react'
+import { focusVisible } from '@vtex/admin-ui-core'
+import { Popover as ReakitPopover } from 'reakit/Popover'
+
+export const Popover = jsx(ReakitPopover)({
+  boxShadow: '$overlay.bottom',
+  border: '$neutral',
+  borderRadius: 'default',
+  bg: '$primary',
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'space-between',
+  minWidth: 200,
+  minHeight: 200,
+  ...focusVisible('neutral'),
+})
+
+export const PopoverFooter = jsx('div')({
+  borderTop: '$neutral',
+  padding: 3,
+  display: 'flex',
+  justifyContent: 'end',
+  'button:not(:first-child)': {
+    marginLeft: 2,
+  },
+})
```

Comment:
You can use the new system for better performance.

```suggestion
export const PopoverFooter = jsx('div')({
  borderTop: '$neutral',
  padding: 3,
  display: 'flex',
  justifyContent: 'end',
  'button:not(:first-child)': {
    marginLeft: 2,
  },
})

export const PopoverFooter = createComponent<typeof Role>(props => {
  return useElement(Role, {
    baseStyle: {
      borderTop: '$neutral',
      padding: 3,
      display: 'flex',
      justifyContent: 'end',
      'button:not(:first-child)': {
         marginLeft: 2,
      },
    },
    ...props,
  })
})
```

PS: Role comes from reakit